dissect/target/plugins/general/loaders.py

```diff
@@ -14,10 +12,8 @@ def check_compatible(self):
     def loaders(self):
         """List the available loaders."""
 
-        loaders = itertools.chain(LOADERS, [DirLoader])
-        loader_names = [loader.attr for loader in loaders]
-        loader_names.sort()
+        loader_names = sorted(getattr(loader, "attr", loader.__name__) for loader in LOADERS + [DirLoader])
 
         print("Available loaders:")
         for loader_name in loader_names:
-            print(f"    {loader_name}")
+            print(f"\t{loader_name}")
```

Comment:
All other output is 4 spaces indented, so this should too. For consistency, it is better to replace it with `dissect.target.helpers.docs.INDENT_STEP`:

```suggestion
from dissect.target.helpers.docs import INDENT_STEP

...

            print(f"{INDENT_STEP}{loader_name}")
```